src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Profile.cshtml

```diff
@@ -0,0 +1,116 @@
+﻿@page "{UserName?}"
+@model APIViewWeb.Pages.Assemblies.ProfileModel
+@using APIViewWeb.Helpers
+@using APIViewWeb.Models
+@{
+    if(Model.User.GetGitHubLogin().Equals(Model.userProfile.UserName)) 
+    {
+        ViewData["Title"] = "Your profile";
+    }
+    else 
+    {
+        ViewData["Title"] = Model.userProfile.UserName;
+    }
+    TempData["UserPreference"] = PageModelHelpers.GetUserPreference(Model._preferenceCache, User.GetGitHubLogin()) ?? new UserPreferenceModel();
```

Comment:
You shouldn't need null coalescing since `GetUserPreference` will never return null https://github.com/Azure/azure-sdk-tools/blob/b009cdc5e2dcaf73d3d9020f4aca1a2a269ad700/src/dotnet/APIView/APIViewWeb/Repositories/UserPreferenceCache.cs#L88